app/controllers/api/v1/access_tokens_controller.rb

```diff
@@ -25,6 +25,14 @@ def deactivate
     render json: { status: "`#{@access_token.owner}` has been deactivated" }.to_json, status: :ok
   end
 
+  def caller_identity
+    if @access_token
+      render json: @access_token.as_json(except: [:token_digest]).to_json, status: :ok
```

Comment:
I can't see any harm in returning the digest for what its worth (its derived from the key they have). During the initial debugging of this new feature we needed to see the digest to see it changing on each invocation, that we got from querying the db directly since its also not returned in the get all endpoint. Going forward it might not be as useful so we could add it if necessary 👍

Reply:
Good point, I've included the token_digest. I excluded it at first because I thought it might be a security risk but if the token being passed in is already known then showing the digest along with the all the other info shouldn't be necessary.